### PR TITLE
EPMDJ-6326: Fix text and layout problems 

### PR DESCRIPTION
**Pull request by IliaSolovev**

### CODE REVIEW

File: src/content-script/components/overlay/overlay.module.scss [hunk 1]
```diff
@@ -2,10 +2,6 @@
 
 .overlay {
   position: absolute;
-  top: 0;
```
Comment (RomanDavlyatshin) [inline src/content-script/components/overlay/overlay.module.scss:5]: Not sure about that one: have you removed that to fix the "Start Testing" error layout?

File: src/content-script/pages/start-new-manual-test/index.tsx [hunk 2]
```diff
@@ -36,7 +36,7 @@ export const StartNewManualTest = () => {
               onChange={({ currentTarget: { value } }) => setTestName(value)}
               disabled={isFormSubmitting}
             />
-            <div title="Enter a test name to start testing">
+            <div title={!testName ? 'Enter a test name to start testing' : undefined}>
```
Comment (RomanDavlyatshin) [inline src/content-script/pages/start-new-manual-test/index.tsx:39]: `testName && 'Enter a test name to start testing'` ? Or we'll get a title with an empty string?

**Review by RomanDavlyatshin — CHANGES_REQUESTED** [review]

File: src/content-script/pages/start-new-manual-test/index.tsx (hunk 2 above)
Comment (IliaSolovev) [reply src/content-script/pages/start-new-manual-test/index.tsx:39]: Ts will show an error that the title can not take `string or boolean`

File: src/content-script/components/overlay/overlay.module.scss (hunk 1 above)
Comment (IliaSolovev) [reply src/content-script/components/overlay/overlay.module.scss:5]: Yes, if I will leave it then the overlay will not take more place than the parent component. But with an error message Error component should have more place.

**Pull request merged**